### PR TITLE
Remove checks_version from the supported attributes

### PR DESCRIPTION
**Pull request by gyfis**

### CODE REVIEW

File: internal/provider/resource_monitor.go [hunk 1]
```diff
@@ -283,65 +281,16 @@ var monitorSchema = map[string]*schema.Schema{
...
-			}
-			return diag.Diagnostics{
-				diag.Diagnostic{
-					AttributePath: path,
-					Severity:      diag.Error,
-					Summary:       `Invalid "ip_version"`,
-					Detail:        fmt.Sprintf("Expected one of %v or nil", ipVersions),
-				},
-			}
-		},
```
Comment (PetrHeinz) [inline internal/provider/resource_monitor.go]: Why do you want to remove the IP version validation? If that was an accident, please revert 🙏

**Review by PetrHeinz — APPROVED** [review]
Thanks! 🙌 Do want to remove the checks_version, and keep only edge-based checks already? Didn't expect we'd be out of beta so soon. 🚀 

You can generate the docs via `make gen`.

**Comment by PetrHeinz** [comment]
I'd also suggest switching to `VERSION := 0.11.0` in the `Makefile` and releasing it as `v0.11.0` after merge.

File: internal/provider/resource_monitor.go (hunk 1 above)
Comment (gyfis) [reply internal/provider/resource_monitor.go]: Right, took that one when I saw `checksVersion` which I assume is a typo; will revert, thx!

I did run `mage gen`, and `make fmt` and the others too 👀 will rerun as well

**Pull request merged**